services/management/checks.go

```diff
@@ -0,0 +1,51 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package management
+
+import (
+	"context"
+
+	"github.com/percona/pmm/api/managementpb"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/percona/pmm-managed/services"
+)
+
+// ChecksServiceAPI represents security checks service API.
+type ChecksAPIService struct {
+	checksService checksService
+}
+
+// NewChecksAPIService creates new Checks API Service.
+func NewChecksAPIService(checksService checksService) *ChecksAPIService {
+	return &ChecksAPIService{checksService: checksService}
+}
+
+// StartSecurityChecks starts STT checks execution.
+func (s *ChecksAPIService) StartSecurityChecks(ctx context.Context, request *managementpb.StartSecurityChecksRequest) (*managementpb.StartSecurityChecksResponse, error) {
```

Comment:
What do you think if we write a few tests for this method? at least that we handle errors correctly

Reply:
Here tests for that method: https://github.com/Percona-Lab/pmm-api-tests/pull/99/files
I don't think there is much sense to duplicate them here.

Reply:
Looks good, bu what about a case when we are unable to download checks or some internal error?

Reply:
Added some tests